enterprise/frontend/src/embedding-sdk/components/private/InteractiveQuestion/context/types.ts

```diff
@@ -69,6 +70,15 @@ type InteractiveQuestionConfig = {
    * A callback function that triggers when a user clicks the back button.
    */
   onNavigateBack?: () => void;
+
+  /**
+   * When provided, this dashboard will be used to navigate back to the dashboard from other view
+   * instead of the state from Redux in `qb.parentDashboard.dashboardId`
+   */
+  backToDashboard?: {
+    id: DashboardId;
+    name: string;
```

Comment:
Nice! I like that it shows the dashboard name as a little tooltip 👍🏻

PS. `backToDashboard` sounds a bit like a `backToDashboard()` function to me.

Reply:
Same remark on the name, but the only one that comes to my mind is `dashboardToGoBackTo` which is really gross, so ¯\_(ツ)_/¯

Reply:
What if we do like `previousDashboard`? What I want to capture in the name here are 2 things:
1. This is the dashboard that we want to go back to from the current screen.
2. This is a dashboard object, not an ID.

`dashboardToGoBackTo` might sound bad, but it's clearer than `backToDashboard` imo, given it sounds like an object rather than a possible function.

Actually, the name is actually trying to follow the pattern we have with `autoScrollToDashcardId`, do you think that's a function or an ID?

Well... it's an ID. And I think that's the same pattern as the `backToDashboard` 🤷 .

I'd probably keep it as is for now after pondering for a while.